src/sentry/web/frontend/base.py

```diff
@@ -469,19 +485,21 @@ def has_permission(self, request: Request, organization, project, *args, **kwarg
         return True
 
     def convert_args(self, request: Request, organization_slug, project_slug, *args, **kwargs):
-        active_organization = self.get_active_organization(
-            request=request, organization_slug=organization_slug
-        )
-
-        if active_organization:
-            active_project = self.get_active_project(
-                request=request, organization=active_organization, project_slug=project_slug
-            )
+        organization: Optional[Organization] = None
+        active_project: Optional[Project] = None
+        if self.active_organization:
```

Comment:
Fortunately, this view subclass is only ued in Region endpoints, I believe, so it's safe to assume it can lookup the ORM objects.